launch/tier4_simulator_launch/launch/simulator.launch.xml

```diff
@@ -128,7 +128,7 @@
   </group>
 
   <!-- Dummy localization -->
-  <group if="$(var launch_dummy_localization)">
+  <group if="$(eval '&quot;$(var localization_sim_mode)&quot;==&quot;api&quot;')">
```

Comment:
It would be better to have a comment indicating that nothing is done when it is 'none'.

Reply:
I added an empty group that does nothing.
https://github.com/autowarefoundation/autoware.universe/pull/8331/files#diff-d52309b103e79aa877d289aade46366674f361ac9c516d846bb01bb2d6eaf62eR134-R136